File: IOT/anshulgurjar53_AnshulSinghGurjar_2024it1193_2/FunctionAndArrays/BarChart.java
import java.io.*;
import java.util.*;

public class Main{

public static void main(String[] args) throws Exception {
    // write your code here
    Scanner sc=new Scanner(System.in);
    int n=sc.nextInt();
    int a[]=new int[n];int max=a[0];
    for(int i=0;i<n;i++)
    {
        a[i]=sc.nextInt();
        if(a[i]>max)
        max=a[i];
    }
    for(int i=0;i<max;i++)
    {
        
        for(int j=0;j<n;j++)
        {
            if((max-a[j])>i)
            System.out.print("\t");
            else
            System.out.print("*\t");
        }
        System.out.println();
    }
 }

}

File: AR-VR/devAyushDubey_ayushdubey_2024csit1122_2/FunctionAndArrays/BarChart.cpp
#include<iostream>
using namespace std;


int main(){
    int n,i,j;
    cin>>n;
    int bar[n],max=0;
    
    for(i=0;i<n;i++){
        cin>>bar[i];
        if(bar[i]>max)
            max=bar[i];
    }
    
    int graph[n][max];
    
    for(i=0;i<n;i++){
        for(j=0;j<max;j++)
            graph[i][j]=-1;
    }
    
    for(i=0;i<n;i++){
        for(j=0;j<bar[i];j++)
            graph[i][j]=0;
    }
    
    for(i=max-1;i>=0;i--){
        for(j=0;j<n;j++){
            if(graph[j][i]==0)
                cout<<"*	";
            else
                cout<<"	";
        }
        cout<<endl;
    }
    //write your code here
    
    
}